### PR TITLE
Renovate: Fix team reviewers

### PR DESCRIPTION
**Pull request by sirreal**
#### Changes proposed in this Pull Request

Add required `team:` prefix to reviewers.

https://docs.renovatebot.com/configuration-options/#reviewers

### CODE REVIEW

**Review by tyxla — APPROVED** [review]
LGTM 🚢

**Review by simison — APPROVED** [review]
Thanks!

File: renovate.json [hunk 1]
```diff
@@ -10,11 +10,7 @@
 			"extends": "monorepo:wordpress",
 			"separateMajorMinor": false,
 			"prPriority": 2,
-			"reviewers": [
-				"@Automattic/cylon",
-				"@Automattic/ganon",
-				"@Automattic/luna"
-			]
+			"reviewers": [ "team:@Automattic/cylon", "team:@Automattic/ganon", "team:@Automattic/luna" ]
```
Comment (blowery) [inline renovate.json]: i'm not sure you need the `@` here...

wish there was a way to test / validate this.

Comment (blowery) [reply renovate.json]: @rarkins what's the right syntax here? we want to use an organization-level team as a reviewer.  should it be `team:combinedSlug` or `team:slug`? Does the `@` matter?

Comment (rarkins) [reply renovate.json]: I'm not sure actually. Here's the code: https://github.com/renovatebot/renovate/blob/0b61cfc17109e0dd4fece6d023eefcaa9e265f40/lib/platform/github/index.ts#L1437-L1451

The GitHub docs are not particularly guiding in this matter: https://developer.github.com/v3/pulls/review_requests/#create-a-review-request

Comment (rarkins) [reply renovate.json]: I would guess you maybe just post `team:slug`. i.e. `team:cylon` on this case.

**Comment by simison** [comment]
Maybe we can just add all 3 formats here and see which team gets the ping
next time? :-D Then docs at Renovate could be updated.

File: renovate.json (hunk 1 above)
Comment (sirreal) [inline renovate.json]:
```suggestion
			"reviewers": [ "team:Automattic/cylon", "team:Automattic/ganon", "team:Automattic/luna" ]
```

**Pull request merged**